algebra/Cargo.toml

```diff
@@ -32,11 +32,12 @@ rand_xorshift = "0.2"
 default = [ "std" ]
 
 full = [
-  "bls12_377", "bls12_381", "cp6_782", "bw6_761", "mnt4_298", "mnt4_753",
+  "bn256", "bls12_377", "bls12_381", "cp6_782", "bw6_761", "mnt4_298", "mnt4_753",
   "mnt6_298", "mnt6_753", "ed_on_bls12_377", "ed_on_cp6_782",
   "ed_on_bw6_761", "ed_on_bls12_381", "ed_on_mnt4_298", "ed_on_mnt4_753"
 ]
 
+bn256 = []
```

Comment:
> Looks great, just some naming suggestions (`bn256` -> `bn254`).

Yeah, I name it after Geth (https://github.com/ethereum/go-ethereum/tree/master/crypto/bn256) although I don't really agree with the name :p

Reply:
Yeah, I don’t know why they renamed it, but in libsnark it’s bn254, so I think being consistent with that would be nice. (Also Zcash used it as the bn254 curve in the past)

Reply:
If I remember correctly, the history goes like this:
* bn256 from cloudflare and google implement a BN curve that's unrelated to Zcash. When the Ethereum devs ported it to the libsnark curve, they kept the name.
* bn128 is the name that was used by libsnark and then by Zcash, denoting 128-bit security.
* Later on, Zcash and the community mostly decided to switch to the bn254 naming, since it wasn't 128-bit security anymore and 254-bit is the size of the curve.

I believe that the pairing-friendly curve draft from IETF actually calls this curve BN_SNARK1.